### ISSUE TITLE
Optimize app

### ISSUE THREAD

**Issue by thibautjombart** [thibautjombart/covid19_bed_occupancy#14]
Current app is slow, but can be optimized, e.g.:

* evaluating reactive items only once (some duplication currently)
* add a "run" button like in [this app](https://github.com/thibautjombart/covid19_cases_from_deaths/tree/master/app)
* consider adjusting limits of some parameters to avoid crazy numbers

**Comment by thibautjombart** [comment]
@pearsonca I dictatorially assigned you this one but please feel free to decline if no time today ;)

**Comment by pearsonca** [comment]
once #19 is merged into master, I'll have a look at speed-ups

**Comment by thibautjombart** [comment]
@pearsonca should be good to go now

**Comment by thibautjombart** [comment]
@pearsonca when you make the changes could you set it up so that the output of `run_model()` sits on its own reactive component, so it can be re-used for plotting and for the table?
